modules/gdscript/gdscript_vm.cpp

```diff
@@ -495,6 +497,11 @@ void (*type_init_function_table[])(Variant *) = {
 #define METHOD_CALL_ON_FREED_INSTANCE_ERROR(method_pointer) "Cannot call method '" + (method_pointer)->get_name() + "' on a previously freed instance."
 
 Variant GDScriptFunction::call(GDScriptInstance *p_instance, const Variant **p_args, int p_argcount, Callable::CallError &r_err, CallState *p_state) {
+	//It would be nice to use something like "ObjectName::function_name"
+	// But it would be much more expensive
+	// eg _script->get_fully_qualified_name() + "::" + name
+	GodotProfileZoneScript(zone, _func_cname, name, source, _initial_line, Color::named("dodger_blue"));
```

Comment:
`Color::named` allocates a `String` for the lookup, which is an unnecessary performance burden for the profile call.
I would remove the `color` parameter for the moment, because it's not strictly necessary to get the feature working.